crates/holochain_sqlite/Cargo.toml

```diff
@@ -33,8 +33,8 @@ once_cell = "1.4.1"
 num_cpus = "1.13.0"
 parking_lot = "0.12"
 r2d2 = "0.8"
-r2d2_sqlite = { version = "0.1", package = "r2d2_sqlite_neonphog" }
-rmp-serde = "=1.1.2"
+r2d2_sqlite = { version = "0.24.0", package = "hc_r2d2_sqlite" }
```

Comment:
Needed a bump and I couldn't work on this fork, switched to one that the core team can update